src/settings/toml/manifest.rs

```diff
@@ -115,6 +120,35 @@ impl Manifest {
         Ok(template_config)
     }
 
+    pub fn worker_name(&self, env_arg: Option<&str>) -> String {
```

Comment:
extracted this method from the `get_target` method for readability and reuse.

Reply:
nit: `worker_name` -> `get_worker_name`